python/shotgun_globals/cached_schema.py

```diff
@@ -738,43 +738,63 @@ def get_status_color(cls, status_code, project_id=None):
         return status_color
 
     @classmethod
-    def field_is_editable(cls, sg_entity_type, field_name):
+    def field_is_editable(cls, sg_entity_type, field_name, project_id=None):
         """
         Returns a boolean identifying the editability of the entity's field.
 
         :param str sg_entity_type: the entity type
         :param str field_name: the field name to check editibility
+        :param project_id:  The id of the project.
+                            If None, the current context's project will be used.
 
         :returns: ``True`` if the field is ediable, ``False`` otherwise.
         """
         self = cls.__get_instance()
-        self._check_schema_refresh(sg_entity_type, field_name)
 
-        if sg_entity_type in self._type_schema and field_name in self._field_schema[sg_entity_type]:
-            data = self._field_schema[sg_entity_type][field_name]
+        project_id = project_id or self._get_current_project_id()
```

Comment:
great! heads up that this is what i am seeing right now w caches:

![image](https://cloud.githubusercontent.com/assets/337710/17057261/d9cf4c08-5010-11e6-8a40-82d2db0d0daa.png)

I *think* with this none change, the `p0` will go away. which is nice!

Cheers all!